pages/edge/quickstart/http-server.mdx

```diff
@@ -21,6 +21,11 @@ The project requires the following tools to be installed on your system:
 
 <Install />
 
+<Callout type="warning" emoji="⚠️">
```

Comment:
We might want to have a shared `CliVersionCallout` component. That way we only need to update the version in one place.

Reply:
made a `CliVersionCallout`